projects/js-packages/publicize-components/src/components/connection-list/list.tsx

```diff
@@ -0,0 +1,43 @@
+import { useCallback } from '@wordpress/element';
+import { Connection } from '../../social-store/types';
+import { ConnectionListItem } from './item';
+import styles from './list-style.module.scss';
+
+export type ConnectionListProps = {
+	connections: Array< Connection >;
+	onToggle: ( connectionId: string ) => void;
+	title: string;
+};
+
+/**
+ * The component to render a list of social media connections.
+ *
+ * @param {ConnectionListProps} props - Component props.
+ * @return {JSX.Element} - React element
+ */
+export function ConnectionList( { connections, onToggle, title }: ConnectionListProps ) {
+	const onConnectionToggle = useCallback(
+		( connectionId: string ) => () => {
+			onToggle( connectionId );
+		},
+		[ onToggle ]
+	);
+
+	return (
+		<section className={ styles.wrapper }>
+			<h4 className={ styles.title }>{ title }</h4>
```

Comment:
@pablinos Like https://github.com/Automattic/jetpack/pull/42221#discussion_r1982093821, do we want to remove the heading from here as well?

Reply:
It doesn't matter so much here, as we're not using it in two places, but maybe we should to future proof it.

It's a prop here, so we could render the title if we have one? We could do the same in the other list too perhaps?

Reply:
Done in d17522b

![CleanShot 2025-03-07 at 09 52 36 png](https://github.com/user-attachments/assets/955b8fe5-8f0a-4462-ae91-38a1d34ef95a)